primo/utils/solvers.py

```diff
@@ -127,15 +129,7 @@ def check_optimal_termination(results, solver):
         Supported solvers include highs, gurobi, scip, and glpk
     """
 
-    if solver in ["glpk", "gurobi", "gurobi_persistent", "scip"]:
-        # This works for Gurobi, SCIP, and GLPK, but not for HiGHS
+    if solver in ["glpk", "gurobi", "gurobi_persistent", "scip", "highs"]:
```

Comment:
Is the check_optimal_termination method still needed after the change to using appsi_highs? Why not use pyo_opt_term directly?

Reply:
I am not sure that I am following. pyo_opt_term is the check_optimal_termination method (see solvers.py::19)